### PR TITLE
added Happy_Number problem's code in java, for issue #1308

### PR DESCRIPTION
**Pull request by rishigarg94**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by rishigarg94** [comment]
@atarax665  @iamrajiv  @plazzy99  please, review my PR for issue #1308  , i have uploaded code in java

File: Java/cp/Happy_Number.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{ 
+    int n ; 
+    System.out.println("Enter number you want to check");
+    Scanner sc = new Scanner(System.in);
+    String inte=sc.next();
+     n = Integer.parseInt(inte);
+
+    if (isHappynumber(n)) 
+        System.out.println(n +  
+        " is a Happy number");
```
Comment (vatsalkesarwani12) [inline Java/cp/Happy_Number.java:56]: refractor your code with proper indentation

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@rishigarg94 Add sample input-output, time complexity and space complexity.

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**